src/main/java/collab/options/first/PrintOption.java

```diff
@@ -3,12 +3,30 @@
 import collab.AbstractFirstOption;
 import collab.Employee;
 
+import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class PrintOption extends AbstractFirstOption {
     @Override
     public String getFilteredList(List<Employee> inputList) {
         // 사번 기준으로 sorting
+        ArrayList<Employee> sortedList = new ArrayList<>();
+        sortedList.addAll(inputList.stream()
+                .sorted(Comparator.comparing(Employee::getRealEmployeeNumber))
+                .limit(5)
+                .collect(Collectors.toList()));
+        if (sortedList.size() != 0 ) return makeResultString(sortedList);
         return null;
     }
+
+    private String makeResultString(List<Employee> sortedList){
+        String result = new String();
+        for (Employee employee: sortedList){
+            result += employee.getEmployeeNumber() + ',' + employee.getName() + ',' + employee.getCareerLevel()
+                    + ',' + employee.getPhoneNumber() + ',' + employee.getBirthday() + ',' + employee.getCerti() + '\n';
+        }
```

Comment:
String 대신 StringBuilder를 써 보는 건 어떨까요?
관련 내용: [https://ifuwanna.tistory.com/221](https://ifuwanna.tistory.com/221)

그리고, \n 대신에 System.lineSeparator()를 쓰면 OS에 관계없이 잘 동작할 것 같습니다.
System.lineSeparator()로 변경하는 것은 어떨까요?
WIndows의 경우는 CRLF인데, 클래식 Max OS에서는 CR, Linux에서는 LF가 개행문자라고 해서요.

```suggestion
        StringBuilder result = new StringBuilder();
        for (Employee employee: sortedList){
            result.append(employee.getEmployeeNumber());
            result.append(",");
            result.append(employee.getName());
            result.append(",");
            result.append(employee.getCareerLevel());
            result.append(",");
            result.append(employee.getPhoneNumber());
            result.append(",");
            result.append(employee.getBirthday());
            result.append(",");
            result.append(employee.getCerti());
            result.append(System.lineSeparator());
        }
        System.out.println(result.toString());
        return result.toString();
```

Reply:
StringBuilder를 사용하면 OS 호환성을 맞춰주는군요!
추후 Refactoring시 반영하도록 하겠습니다!

Reply:
앗. StringBuilder는 String이 +로 추가되는 것이 느린 것을 보완하기 위해서 사용하는 것으로 알고 있어요.
OS 호환성 이야기는 System.lineSeparator()에만 해당되는 이야기예요.